support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/heading.tsx

```diff
@@ -193,13 +197,21 @@ function Heading({
 	amount,
 	currency,
 	contributionType,
+	ratePlanKey,
 	paymentStatus,
 	promotion,
 }: HeadingProps): JSX.Element {
 	const isPending = paymentStatus === 'pending';
 	const isDigitalEdition = productKey === 'DigitalSubscription';
 	const isGuardianAdLite = productKey === 'GuardianAdLite';
 	const isTier3 = productKey === 'TierThree';
+	const isPaperOrGuardianWeekly = [
```

Comment:
This comment got lost in an update:

Rather than defining this here it might be better to get it from the productDescription like we do on the checkout.

https://github.com/guardian/support-frontend/blob/9d6f65e4ec8a0de4d3d727b8af980931d19d9d2a/support-frontend/assets/pages/%5BcountryGroupId%5D/components/checkoutComponent.tsx#L185

If you need something which isn't there currently it could be added as a new field and then we could just pull back the relevant info by productKey rather than having to have all these if clauses.

Reply:
Can see using `productCatalog` would be a cleaner approach dependent on how much dynamic content we need to display. As George mentioned in standup, we will look into this later.